.github/workflows/avm.template.module.yml

```diff
@@ -132,6 +133,9 @@ jobs:
           customLocation: "${{ fromJson(inputs.workflowInput).customLocation }}"
         env:
           AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+          VALIDATE_CLIENT_ID: ${{ secrets.VALIDATE_CLIENT_ID }}
+          VALIDATE_SUBSCRIPTION_ID: ${{ secrets.VALIDATE_SUBSCRIPTION_ID }}
+          VALIDATE_TENANT_ID: ${{ secrets.VALIDATE_TENANT_ID }}
```

Comment:
Do I understand this correctly that we now have both a secret `VALIDATE_SUBSCRIPTION_ID` and `ARM_SUBSCRIPTION_ID` with the same value? Would not be one of the other be better? Not to mention an aligned naming pattern? :)

Reply:
This secret duplication is only temporary to avoid breaking changes for contributors who already have a testing environment set up. 
 After an agreed grace period we should only use the VALIDATE* ones and update the token replacement accordingly.
Also please note that the VALIDATE* once are set at gh environment level.
The naming came from the fact that in BRM we also have PUBLISH* ones. This differentiation may come useful for the innersource topic as well

Reply:
Sounds fair, and if it's only temporarily / for testing it doesn't matter too much anyways. Just curious